wordpress-mobile/WordPress-iOS
# Reader post comments: show first comment

Ref: #17511 
Depends on:  #17602

The first comment is now displayed in the Reader post detail Comments table.

Note: the `postDetailsComments` feature flag is enabled for develop.

To test:
- Go to Reader > post > details.
- Verify:
  - The first comment is displayed in the Comments table.
  - All the comment actions are hidden (like, reply, moderation, accessory button)

<kbd><img width="470" alt="comment" src="https://user-images.githubusercontent.com/1816888/144139218-a81f3390-1f9a-40a2-8607-7aadfdef7caf.png"></kbd>

## Regression Notes
1. Potential unintended areas of impact
N/A. WIP.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A. WIP.

3. What automated tests I added (or what prevented me from doing so)
N/A. WIP.

PR submission checklist:
